contrib/sqlalchemy-connector/README.md

```diff
@@ -0,0 +1,17 @@
+### Superset usage
```

Comment:
This connector may not only be used by supserset. It may also be used as a general client for python to connect to sr to write code. But it might be alright to write that first.

Reply:
README.md Optimized.